bindata/network/openshift-sdn/000-ns.yaml

```diff
@@ -9,3 +9,16 @@ metadata:
   annotations:
     openshift.io/node-selector: "" #override default node selector
     openshift.io/description: "OpenShift SDN components"
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  # NOTE: this namespace is the portal for host network traffic
+  name: openshift-host-network
+  labels:
+    openshift.io/run-level: "0"
+    openshift.io/cluster-monitoring: "true"
+    network.openshift.io/policy-group: "host-network"
+  annotations:
+    openshift.io/node-selector: ""
+    openshift.io/description: "Namespace for enabling network policy specification for host network traffic on OpenShift SDN"
```

Comment:
Suggestion: "Placeholder namespace for host-network traffic in NetworkPolicy. Can be used to allow access to or from host network components."

@jboxman any thoughts?